web/style.css

```
@@ -160,7 +172,13 @@ body {
   position: absolute; width: 1px; height: 1px; margin: -1px; padding: 0;
   overflow: hidden; clip-path: inset(50%); white-space: nowrap; border: 0;
 }
-.block { padding: 16px 18px; border-bottom: 1px solid var(--line-2); }
+/* 余白は節の中身だけでなく、行の上に重ねる釦の置き場所も決める(#sel-none)。
+   同じ数を二箇所に書かないよう、変数にして下の者へ渡す。 */
+.block {
+  --pad-y: 10px; --pad-x: 18px;
+  position: relative; padding: var(--pad-y) var(--pad-x);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)